src/Common/nunit/CommandLineOptions.cs

```diff
@@ -40,7 +40,9 @@ namespace NUnit.Common
     public class CommandLineOptions : OptionSet
     {
         private bool validated;
+#if !NETCORE
```

Comment:
I agree, same for NUnitLite. My plan was to start small and once we have time, create a more full featured runner. I'm still not sure if it will have to be a full console runner, an nunitlite type runner or something else. So far, I am adding an nunitlite type runner.